src/YoutubeView.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+
+import logo from './assets/logo.png';
+import './styles/App.css';
+import { useYoutubeAuth } from './contexts/YoutubeContext';
+import { generateYoutubeUrl } from './services/Youtube';
+import { Playlist } from './types/Playlist';
+import { PlaylistQuery } from './types/PlaylistQuery';
+import { TOKEN_YOUTUBE_TYPE, TokenMessage } from './types/TokenTypes';
+import OpenAuthenticationPopup from './utils/openAuthenticationPopup';
+
+function YoutubeView(): JSX.Element {
```

Comment:
```suggestion
export const YoutubeView = (): JSX.Element => {
```

I consider more cleaner to use arrow functions